internal/srv/handlers.go

```diff
@@ -47,12 +47,18 @@ func (s *Server) MessageHandler(m *nats.Msg) {
 func (s *Server) createMessageHandler(m *pubsubx.Message) error {
 	lbdata := events.LoadBalancerData{}
 
+	urn, err := urnx.Parse(m.SubjectURN)
```

Comment:
I think a comment that is this is the URN of the LB would be useful. At first glance I assumed we went from `$LB_URN` to `$SOME_URN`